lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -273,21 +273,6 @@ sub name {
     return $self->{_name};
 }
 
-sub label {
-    my ($self) = @_;
-
-    my $test    = $self->TEST;
-    my $machine = $self->MACHINE;
-    return $machine ? "$test\@$machine" : $test;
-}
-
-sub scenario {
```

Comment:
As far as I remember this is used for the scenario description so there should be *some* replacement.